frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -263,6 +253,28 @@ const DeviceUserPage = ({
   } = dupResponse || {};
   const isPremiumTier = license?.tier === "premium";
 
+  const isDarwinHost = host?.platform === "darwin";
+  const isIosOrIpadosHost =
+    host?.platform === "ios" || host?.platform === "ipados";
```

Comment:
Consider using this utility [function](https://github.com/fleetdm/fleet/blob/1a1c4923d7e382ac4cba57049b02478a70f8b9c1/frontend/interfaces/platform.ts#L131-L132)